src/rust-crypto/libolm_migration.ts

```diff
@@ -158,7 +160,17 @@ async function migrateBaseData(
     // If we have a backup recovery key, we need to try to figure out which backup version it is for.
     // All we can really do is ask the server for the most recent version.
     if (recoveryKey) {
-        const backupInfo = await requestKeyBackupVersion(http);
+        let backupCallDone = false;
+        let backupInfo: KeyBackupInfo | null = null;
+        while (!backupCallDone) {
+            try {
+                backupInfo = await requestKeyBackupVersion(http);
+                backupCallDone = true;
+            } catch (e) {
```

Comment:
Should we log here? Is this a fully-expected error? I'd be more comfortable if we could see some information about `e` in the log, in case something we didn't expect happens.

Reply:
Added a log line